### DISCUSSION TITLE
Document template function

### DISCUSSION THREAD

**Discussion by KurtMoan** [steven-tey/novel#224] [Q&A]
Is it possible to easily create a template function?

With that I mean create predefined documents (templates) with a certain structure for users to choose amongst.
I.e. Meeting template, Specific project templates, Report tamplate, etc.

**Comment by arthureberledev** [comment]
You could set templates with the `defaultValue` prop. Best way probably would be to predefine the templates by writing them into the editor and save the output manually through the `editor.getJSON()` function
